### PR TITLE
Nh/remove usresync crypto

### PR DESCRIPTION
**Pull request by nhachicha**
Remove code related to secure `SynUser` token, as now it lives in a separate repo https://github.com/realm/realm-android-user-store.

This PR also remove `Twitter` as an OAuth provider, as it's not supported by ROS.

### CODE REVIEW

**Comment by nhachicha** [comment]
@realm/java now ready to review. The binaries are in JCenter

File: examples/secureTokenAndroidKeyStore/build.gradle [hunk 1]
```diff
@@ -30,6 +30,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:24.2.0'
     testCompile 'junit:junit:4.12'
+    compile 'io.realm:secure-userstore:1.2.0'
```
Comment (cmelchior) [inline examples/secureTokenAndroidKeyStore/build.gradle]: Already 1.2.0? 😄 
It is possible to delete releases on Bintray and reupload them. Minor issue though

File: examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/securetokenandroidkeystore/MainActivity.java [hunk 2]
```diff
@@ -30,18 +30,19 @@
...
+import io.realm.SyncManager;
+import io.realm.SyncConfiguration;
+import io.realm.Realm;
 import io.realm.internal.objectserver.Token;
+import io.realm.internal.objectserver.ObjectServerUser;
 
 /**
  * Activity responsible of unlocking the KeyStore
- * before using the {@link io.realm.android.SecureUserStore} to encrypt
+ * before using the {@link realm.io.android.SecureUserStore} to encrypt
```
Comment (cmelchior) [inline examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/securetokenandroidkeystore/MainActivity.java]: This doesn't look right?

File: examples/secureTokenAndroidKeyStore/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -3,7 +3,8 @@
     package="com.example.securetokenandroidkeystore">
 
     <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name" android:supportsRtl="true" android:theme="@style/AppTheme">
+        android:label="@string/app_name" android:supportsRtl="true" android:theme="@style/AppTheme"
+        android:name="examples.io.realm.securetokenandroidkeystore.securetokenandroidkeystore.MyApplication">
```
Comment (cmelchior) [inline examples/secureTokenAndroidKeyStore/src/main/AndroidManifest.xml]: No need for two `securetokenandroidkeystore` packages?

File: examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/securetokenandroidkeystore/MyApplication.java [hunk 4]
```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-package io.realm.internal.android.crypto.misc;
+package examples.io.realm.securetokenandroidkeystore.securetokenandroidkeystore;
```
Comment (cmelchior) [inline examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/securetokenandroidkeystore/MyApplication.java]: Same? Shouldn't this be `io.realm.examples.securetokenandroidkeystore`

**Comment by nhachicha** [comment]
PR updated @realm/java

File: examples/secureTokenAndroidKeyStore/build.gradle [hunk 5]
```diff
@@ -30,6 +30,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:24.2.0'
     testCompile 'junit:junit:4.12'
+    compile 'io.realm:secure-userstore:1.0.0'
```
Comment (beeender) [inline examples/secureTokenAndroidKeyStore/build.gradle]: do we need an `android` in the lib name like `io.realm:android-secure-userstore`

Comment (Zhuinden) [inline examples/secureTokenAndroidKeyStore/build.gradle]: That makes sense if you look at "realm-android-adapters"

Comment (nhachicha) [inline examples/secureTokenAndroidKeyStore/build.gradle]: I don't mind renaming it, although we already discussed naming in #java WDYT @realm/java

Comment (nhachicha) [inline examples/secureTokenAndroidKeyStore/build.gradle]: ok pushed `android-secure-store` to Bintray waiting for the package to make it to JCenter

**Comment by nhachicha** [comment]
@realm/java `android-secure-userstore` is published by Bintray to JCenter, PR ready to be merged.

File: examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/MainActivity.java [hunk 6]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package examples.io.realm.securetokenandroidkeystore.securetokenandroidkeystore;
+package examples.io.realm.securetokenandroidkeystore;
```
Comment (dalinaum) [inline examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/MainActivity.java]: `package examples.io.realm.securetokenandroidkeystore;` seems little weird.
How about `package io.realm.securetokenandroidkeystore.examples;`?

Comment (cmelchior) [inline examples/secureTokenAndroidKeyStore/src/main/java/examples/io/realm/securetokenandroidkeystore/MainActivity.java]: Yup. I think that is pattern the other examples follow

**Comment by nhachicha** [comment]
@cmelchior @dalinaum fixed the package name

**Pull request merged**